mixer/adapter/svcctrl/template/svcctrlreport/template.proto

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017 Istio Authors
```

Comment:
I believe the mandate now is to commit generated code for protos along side the protos themselves (@mandarjog to confirm).  Can you add in the template.pb.go to the PR (or in a separate template-focused PR) ?

Reply:
It doesn't seem to needed for the build. I added it back anyway.